### PR TITLE
Update graph ops parser & cleanup

### PR DESCRIPTION
**Pull request by alifahrri**
## Type of changes

 

Please check the type of change your PR introduces:
- [x] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, renaming)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] Documentation content changes
- [ ] Other (please describe): 

## What is the current behavior?

- graph ops using third party apps to do parsing

Related issues : 

## What is the new behavior?

- graph ops using builtin json to do parsing
-
-

## Checklist

- [x] I have added tests that prove my fix is effective or that my feature works
- [x] I have added necessary documentation (if appropriate)
- [x] Any dependent changes have been merged and published in downstream modules
- [x] I have filled the `[Unreleased]` [Changelog](https://github.com/nodefluxio/vortex/blob/master/CHANGELOG.md)

### CODE REVIEW

**Comment by triwahyuu** [comment]
for the CI, I think it is better to separate the linter and testing, since testing only have been set up, probably you could delete the testing phase in "Lint and Test" action.

File: .github/workflows/lint.yml [hunk 1]
```diff
@@ -1,17 +1,11 @@
...
-on: [pull_request]
+on:
+  push:
+    branches: [master]
+  pull_request:
+  release:
+    types: [published, edited]
 
 jobs:
   tests:
```
Comment (triwahyuu) [inline .github/workflows/lint.yml:11]: probably you could rename the job to `lint` for the correct description

**Comment by triwahyuu** [comment]
other than that, LGTM

File: .github/workflows/lint.yml (hunk 1 above)
Comment (alifahrri) [reply .github/workflows/lint.yml:11]: oh right thanks

**Review by triwahyuu — APPROVED** [review]

**Comment by alifahrri** [comment]
thanks @triwahyuu

**Pull request merged**